apps/end-tenancy/acceptance/features/tenant-details.js

```diff
@@ -18,6 +18,13 @@ Scenario('I am taken to the name substep when I arrive on the page', (
   I.seeInCurrentUrl(tenantDetailsPage.name.url);
 });
 
+Scenario('I dont see a delete button on page load', (
```

Comment:
implicitly yes - there is a test that adds a tenant and deletes it again. This would throw an error if the button didn't exist